alicloud/resource_alicloud_cdn_domain.go

```diff
@@ -389,6 +415,18 @@ func resourceAlicloudCdnDomainUpdate(d *schema.ResourceData, meta interface{}) e
 		}
 	}
 
+	if d.HasChange("https_config") {
+		if d.IsNewResource() {
+			err := WaitForDomainStatus(d.Id(), Online, 360, meta)
+			if err != nil {
+				return fmt.Errorf("Timeout when Cdn Domain Online")
```

Comment:
Please print err details before return err.

Reply:
This behavior was copied from https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/resource_alicloud_cdn_domain.go#L295. Should that part of the provider also be changed?

Reply:
yes